lib/data-types.js

```diff
@@ -503,6 +503,22 @@ DATEONLY.prototype.toSql = function() {
 };
 
 /**
+ * A timestamp column. Only available in mssql.
```

Comment:
Yeah, It looks like this "TIMESTAMP" is not exactly a timestamp, it a time based sequential number. Its better to have different name so it doesn't confuse.

We will go with "ROWVERSION", as per your PR description we still need `timestamp` name in SQL queries, `toSQL` can handle that

Still we need to move it MSSQL code base, because other dialect doesn't support "ROWVERSION", but they do support "TIMESTAMP" which we can introduce later on